elated-egrets/bangazon-internal-site
# refactored model to just one table for training programs

## DESCRIPTION
Changed training program model to be one resource table

## How to pull down
Commands

```
git fetch --all
git checkout rm-refactor-training-programs-model
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. Remake migrations and make sure correct table was added to database

## What you changed
Took the old tables for training programs and merged the fields into one table per our conversation with joe about the best way to start out.

## What was added to read me
Edited resource description in readme to reflect changes

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
